crates/gen-guest-c/src/lib.rs

```diff
@@ -1623,7 +1626,7 @@ impl Bindgen for FunctionBindgen<'_, '_> {
                 for op in operands {
                     uwriteln!(result, "{},", op);
                 }
-                result.push_str("}");
+                result.push('}');
```

Comment:
I don't think these substitutions of push_str("_") to push('_') are worthwhile, they make it less consistent to read.